provenance fix when reference is inaccessible, now ignores it.

eapearson opened the pull request:
Even though some of the code ignored this case, other code would 
prepare invalid data for d3 which would throw an error and leave an
empty graph.

briehl commented:
Huh. Build is failing since it doesn't have a proper .travis.yml to build with (it's trying to use `rake` and I'm pretty sure this isn't written in Ruby).

Merging anyway.

eapearson committed:
provenance fix when reference is inaccessible, ignores it.
Even though some of the code ignored this case, other code would 
prepare invalid data for d3 which would throw an error and leave an
empty graph.
